airflow/dags/unzip_and_validate_gtfs_schedule/unzip_gtfs_schedule.py

```diff
@@ -117,7 +117,7 @@ def process_feed_files(
     md5hash = hashlib.md5()
     if not is_valid:
         raise ValueError(
-            "Unparseable zip: File/directory structure within zipfile cannot be unpacked"
+            "Unparsable zip: File/directory structure within zipfile cannot be unpacked"
```

Comment:
I feel a bit iffy about changing this spelling in an error where people may try to do string matching to query. Wiktionary lists it as an alternate spelling: https://en.wiktionary.org/wiki/parseable